### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server-Rewrite Maps_Screen Reader: NVDA is not reading about OK button after editing the text field. It's reading only Cancel and Delete button

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#109]
**Environment Details:**
URL:https://manage.iis.net/webserver/certificates
Browser Details:  Google Chrome Version 68.0.3440.106
Screen Reader: NVDA
**MAS ID Violated:**  MAS1.3.2

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till Web Server and press enter
5) Tab till Default Documents and press enter
6) Start Screen Reader (Narrator/NVDA).
7) Verify that screen reader is reading in such a way that meaning of the information isn't lost throughout the process.

**Actual Result:**
NVDA is not reading about OK button after editing the text field. It's reading only Cancel and Delete button 

**Expected Result:**
Screen Reader should read the information in correct order throughout the process.

**User Impact:** 
Visually Impaired Users may face difficulty in navigating through the contents and controls if screen reader does nor read the controls in proper sequence

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}

**Attachments:**
[Bug_109_NVDA is not reading about the OK button.pptx](https://github.com/Microsoft/IIS.WebManager/files/2323846/Bug_109_NVDA.is.not.reading.about.the.OK.button.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
Fixed

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.2; #C+E-H; #.NETWebsites; #Chrome; #NVDA

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev1;

**Comment by sspatil123** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will  be removed.
